src/components/fundsInStake.tsx

```diff
@@ -0,0 +1,104 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
+import {
+  Box,
+  Button,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverCloseButton,
+  PopoverContent,
+  PopoverFooter,
+  PopoverHeader,
+  PopoverTrigger,
+  Portal,
+  Text
+} from '@chakra-ui/react'
+import React, { useMemo } from 'react'
+import { textStyles } from '../constants'
+import { useExtension } from '../states/extension'
+import { hexToFormattedNumber, hexToNumber } from '../utils'
+import { TooltipAmount } from './tooltipAmount'
+
+interface ActionsProps {
+  operatorId: string
+}
+
+export const FundsInStake: React.FC<ActionsProps> = ({ operatorId }) => {
+  const { chainDetails, stakingConstants } = useExtension((state) => state)
+
+  const operator = useMemo(
+    () => stakingConstants.operators.find((operator) => operator.operatorId === operatorId),
+    [stakingConstants.operators, operatorId]
+  )
+
+  const nominatorsStake = useMemo(
+    () =>
+      operator &&
+      stakingConstants.nominators
+        .filter(
+          (nominator) => nominator.operatorId === operatorId && nominator.nominatorOwner != operator.operatorOwner
+        )
+        .reduce((acc, nominator) => acc + hexToNumber(nominator.shares), 0),
```

Comment:
Does this give back shares or nominator value? I think the value would be shares*(totalStake/totalShares) but I'm not 100% sure.

Reply:
it was returning the share, but I fix it to calculate the amount from that share and totalShares in operators